cmd/build.go

```diff
@@ -73,28 +71,35 @@ func cmdBuild(cmd *cli.Cmd) {
 				logrus.Fatalf("Generating graph failed: %v", err)
 			}
 		}
-	}
+	},
 }
 
-func getWorkingDir() (string, error) {
-	currentDir, err := os.Getwd()
-	if err != nil {
-		return "", fmt.Errorf("failed to get current working directory: %w", err)
-	}
-	return currentDir, nil
+func init() {
+	rootCmd.AddCommand(buildCmd)
+
+	buildCmd.Flags().Bool(keyDryRun, false, "Simulate what would happen without actually doing anything dangerous.")
+	buildCmd.Flags().Bool(keyForceRebuild, false, "Forces rebuilding the entire image graph, without regarding if the target version already exists.") //nolint:lll
+	buildCmd.Flags().Bool(keyDisableGraph, false, "Disable generation of graph during the build process.")                                             //nolint:lll
+	buildCmd.Flags().Bool(keyDisableTests, false, "Disable execution of tests during the build process.")                                              //nolint:lll
+	buildCmd.Flags().Bool(keyDisableJUnit, false, "Disable generation of junit reports when running tests")
+	buildCmd.Flags().Bool(keyRetagLatest, false, "Should images be retagged with the 'latest' tag for this build") //nolint:lll
+	buildCmd.Flags().Bool(keyLocalOnly, false, "Build docker images locally, do not push on remote registry")
+	buildCmd.Flags().StringP(keyBackend, "b", backendDocker, fmt.Sprintf("Build backend used to run image builds. Supported backends: %v", supportedBackends)) //nolint:lll
+
+	_ = viper.BindPFlags(buildCmd.Flags())
 }
 
-func doBuild(opts buildOpts) (*dag.DAG, error) {
+func doBuild() (*dag.DAG, error) {
```

Comment:
I'm not a big fan of this change. I liked passing a struct that holds the configuration so it was decoupled from the `flag` implementation (`mow.cli` or `viper`).
Decoupling makes it easier to add tests, and tests wouldn't need to change if we change the way we load configuration.
Also, `viper.GetString(...)` is harder to read IMO